keylime/cloud_verifier_tornado.py

```diff
@@ -677,6 +680,7 @@ def post(self):
 
                     # Write the agent to the database, attaching associated stored policy
                     try:
+                        assert runtime_policy_stored
```

Comment:
The session.add() seems to require it nut there's `runtime_policy_stored = None` right above ...